tests/test_C0_data_processing.py

```diff
@@ -1080,7 +1083,42 @@ def test_change_sign_of_feedin_tariff_zero(caplog):
     ), f"A msg is logged although the feed-in tariff is not changed."
 
 
+def test_compute_timeseries_properties_TIMESERIES_in_dict_asset():
+    str = "str"
+    dict_asset = {
+        TIMESERIES: pd.Series([10, 50, 100, 150, 200]),
+        UNIT: "str",
+        LABEL: "str",
+    }
+
+    C0.compute_timeseries_properties(dict_asset)
+    print(dict_asset)
+
+    for parameter in [
+        TIMESERIES_PEAK,
+        TIMESERIES_TOTAL,
+        TIMESERIES_AVERAGE,
+        # TIMESERIES_NORMALIZED,
+    ]:
+        assert (
+            parameter in dict_asset
+        ), f"Parameter {parameter} is not in updated dict_asset."
+        assert VALUE in dict_asset[parameter]
+        assert UNIT in dict_asset[parameter]
+
+    assert dict_asset[TIMESERIES_PEAK][VALUE] == 200
+    assert dict_asset[TIMESERIES_TOTAL][VALUE] == 510
+    assert dict_asset[TIMESERIES_AVERAGE][VALUE] == 102
+    exp = pd.Series([0.05, 0.25, 0.5, 0.75, 1])
+    assert dict_asset[TIMESERIES_NORMALIZED] == exp.all()
```

Comment:
```suggestion
    assert (dict_asset[TIMESERIES_NORMALIZED] == exp).all()
```
I think the way to do this is. (Comp. [mahendras code](https://github.com/rl-institut/multi-vector-simulator/blob/149e3982879ed224874d1bcacffcadbce8226adb/src/multi_vector_simulator/E4_verification.py#L194)).

This would probably also work with what you previously had, ie.
```
assert (dict_asset[TIMESERIES_NORMALIZED].values == [0.05,0.25,0.5,0.75,1]).all()
```
Choose either one that runs though :)